nova_vm/src/ecmascript/builtins/number.rs

```diff
@@ -224,4 +265,68 @@ impl NumberConstructor {
         // 5. Set O.[[NumberData]] to n.
         // 6. Return O.
     }
+
+    /// ### [21.1.2.2 Number.isFinite ( number )](https://tc39.es/ecma262/#sec-number.isfinite)
```

Comment:
**thought:** I had been using the same number of `#` characters as there are "levels" in the spec definition number, so `21.1.2.2` would be `####`.

That being said, it's not necessarily a very logical system so I definitely do not mind it not being used :D

Reply:
Yeah, I realized the meaning after I saw two headers being used. Although, I like the look of how `###` looks in the code and in intellisense as an `h3`. Do you think it would be all right if we just used `###` for everything, and people can use the numbering to understand the level?

Reply:
Yeah, sure I guess.

I'd like to keep the module level headers different though, maybe.